go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/jetstack/preflight
 
-go 1.22.0
+go 1.23.0
+
+toolchain go1.23.4
```

Comment:
The `toolchain` stanza keeps re-appearing after running go mod tidy with the vendored version of Go:

```console
$ make vendor-go
$ _bin/tools/go version  
go version go1.23.4 darwin/arm64
$ make go-tidy
```